.github/workflows/pull-request.yml

```
@@ -0,0 +1,666 @@
+            echo "any_changed=true" >> $GITHUB_OUTPUT
+          else
+            echo "any_changed=false" >> $GITHUB_OUTPUT
+          fi
+
+          # Set only_changed to false by default (adjust logic as needed)
+          echo "only_changed=false" >> $GITHUB_OUTPUT
+
+      - name: Check formatting
+        if: steps.changed-files.outputs.only_changed != 'true'
+        run: npm run format:check
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)